assets/js/util/ModuleCategory.js

```diff
@@ -0,0 +1,17 @@
+import { t } from '@lingui/macro'
+
+const CategoryNames = {
+  Combine: t('js.util.ModuleCategory.CategoryNames.Combine')`Combine`,
+  Scrape: t('js.util.ModuleCategory.CategoryNames.Scrape')`Scrape`,
+  Clean: t('js.util.ModuleCategory.CategoryNames.Clean')`Clean`,
+  Analyze: t('js.util.ModuleCategory.CategoryNames.Analyze')`Analyze`,
+  Visualize: t('js.util.ModuleCategory.CategoryNames.Visualize')`Visualize`,
+  Code: t('js.util.ModuleCategory.CategoryNames.Code')`Code`,
+  'Add data': t('js.util.ModuleCategory.CategoryNames.AddData')`Add data`,
+  Other: t('js.util.ModuleCategory.CategoryNames.Other')`Other`
+}
+
+export function getCategoryName (i18n, category) {
```

Comment:
I find this pattern unpleasant -- not very React-like.

Would this work better as a `withI18n()` React component? Like:

```javascript
const CategoryName = ({ id }) => {
  switch (id) {
    "Combine": return <Trans ...>;
    "Scrape": return <Trans ...>;
    ....
  }
}
```

Would this look nicer from the caller's point of view? (I'm not 100% sure it would be better, but I think it's worth checking.)

Reply:
It would be nicer in some cases, but it's not always feasible: for example, if it were used in an HTML attribute, it would need to be a plain string; also, `ImportModuleForGithub` needs it to be a string (I tried using a react component and got [Object object] in the status message). 

I am adding a React component for use when appropriate, but I will also keep the `getCategoryName` function for the other cases.

Reply:
Thanks for exploring!

I see no value in an unused function -- especially since it'll always be in Git, forever :). But I'll merge anyway.